apps/dotcms-ui/src/app/api/services/dot-configuration/dot-configuration.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { CoreWebService } from '@dotcms/dotcms-js';
+import { map, pluck, take } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DotConfigurationService {
+    constructor(private coreWebService: CoreWebService) {}
+
+    /**
+     * Get the value of specific key
+     * from the dotmarketing-config.properties
+     *
+     * @returns {Observable<T>}
+     * @memberof DotConfigurationService
+     */
+    getKey<T>(key: string): Observable<T> {
```

Comment:
Why `T` if you already know what this endpoint return?

Reply:
assuming that can be any time, after check with Sanchez will only strings.